samples/Calling/src/app/App.tsx

```diff
@@ -58,7 +60,16 @@ const App = (): JSX.Element => {
   const [userCredentialFetchError, setUserCredentialFetchError] = useState<boolean>(false);
 
   // Call details to join a call - these are collected from the user on the home screen
-  const [callLocator, setCallLocator] = useState<CallAdapterLocator>(createGroupId());
+  const [callLocator, setCallLocator] = useState<CallAdapterLocator>();
+  const [targetCallees, setTargetCallees] = useState<
+    (
+      | MicrosoftTeamsAppIdentifier
```

Comment:
this typing is used in so many places, can we define a `type` or `interface` somewhere for this